src/js-form-validator.ts

```diff
@@ -65,6 +67,9 @@ export class FormValidator {
     else {
       if (allowEmpty && !inputValue.length)
         valid = 1;
+      else if (!this.HasValidLength(inputValue, parseInt(minLength, 10), parseInt(maxLength, 10))) {
```

Comment:
Seems like this could forego using brackets.

Reply:
@samiam236 good catch